sw/device/boot_rom/irq_vector.S

```diff
@@ -0,0 +1,61 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * Interrupt handler vector, set up so that all interrupts are 
+ * caught and loop forever.
+ *
+ * See the Ibex manual, Chapter 11, for more information.
+ */
+  // NOTE: The "ax" flag below is necessary to ensure that this section 
+  // is allocated space in ROM by the linker.
+  .section .vectors, "ax"
+
+  // Disable RISC-V instruction compression: we need all instructions to 
+  // be exactly word wide in the interrupt vector.
+  .option norvc
+
+  // Exception handler.
+  .org 0x00
+  j exception_handler
+
+  // Software interrupt handler.
+  .org 0x0c
+  j default_irq_handler
+
+  // Timer interrupt handler.
+  .org 0x1c
+  j default_irq_handler
+
+  // External interrupt handler
+  .org 0x2c
+  j default_irq_handler
+
+  // Fast interrupts, 0 through 14.
+  .org 0x40
+  .rept 14
+  nop
```

Comment:
I know this code is copied verbatim, but can we document that we explicitly want these 14 nops to fallthrough to `default_irq_handler`?